templates/patterns/featured_item/pattern-featured-item.html.twig

```diff
@@ -1,13 +1,49 @@
 {#
 /**
  * @file
- * Field.
+ * Featured item pattern.
  */
 #}
-{% include '@ecl/ec-component-featured-item' with {
-  'variant': variant,
-  'image': image,
-  'metas': metas,
-  'title': title,
-  'link': link,
+
+{#
+  In order to replicate the extended functionality we add a button
+  at the end of the description if the variant is set to "extended".
+ #}
+{% if variant == 'extended' %}
+  {% set _description %}
```

Comment:
It looks like a leftover.

Reply:
We are setting here the value of `_description`.

Reply:
![image](https://user-images.githubusercontent.com/153362/68191895-c4a53880-ffb0-11e9-878b-ce275d0a0737.png)

Reply:
yes. sorry. didn't know this.